SpatialGDK/Documentation/content/map-travel.md

```diff
@@ -0,0 +1,120 @@
+# [Map travel](https://docs.unrealengine.com/en-us/Gameplay/Networking/Travelling)
+
+> Note: This doc is intended for advanced users only.
+
+## `APlayerController::ClientTravel`
+
+### In native Unreal
+`ClientTravel` is the process of changing which [map (or Level - see Unreal documentation)](http://api.unrealengine.com/INT/Shared/Glossary/index.html#l) a client currently has loaded.
+
+### User guide
+In the GDK you can use `ClientTravel` to move a client-worker from an offline state to a connected state, whether that be connected to a local deployment or a cloud deployment. Alternatively, you can move a connected client-worker to an offline state. You can also change which SpatialOS deployment the client-worker is connected to. 
```

Comment:
"whether that be" is quite a UK English phrase. 

I'd suggest

"to a connected state, where that connection is to a local deployment or a cloud deployment."

Or something similar.